scripts/third_party_size_check.py

```diff
@@ -33,7 +33,7 @@
 import yaml  # isort:skip  #pylint: disable=wrong-import-position
 
 THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party')
-THIRD_PARTY_SIZE_LIMIT = 12000
+THIRD_PARTY_SIZE_LIMIT = 13500
```

Comment:
What's the upper limit on the number of files that can be uploaded on the server? (Just wanted to make sure it doesn't cross that limit)

Reply:
@seanlip ?

Reply:
15k, I think.

Reply:
Should we just change it to 15000 now and stop allowing updates to the value?

Reply:
No, keep it as low as possible. I think this is just a temporary change while we sort out the migration.